ballerina/tests/websub_publisher_test.bal

```diff
@@ -126,6 +199,9 @@ public function testPublisherPubishEventFailure() {
     if response is UpdateMessageError {
         CommonResponse details = response.detail();
         test:assertEquals(details.statusCode, http:STATUS_OK);
```

Comment:
Why the `UpdateMessageError` provides 200 status code? Shouldn't it be error response code in 5XX range

Reply:
In WebSubHub API, default status code for `websubhub:UpdateMessageError` is HTTP 200 OK (this has been used with almost all the errors). We need to refine this status code usage in it, but it will break the already existing code bases. Since, it is a different issue thought to keep out from this PR and created following issue [1] to track it.

[1] - [https://github.com/ballerina-platform/ballerina-standard-library/issues/3003](https://github.com/ballerina-platform/ballerina-standard-library/issues/3003)